### DISCUSSION TITLE
VPC Peering Connection

### DISCUSSION THREAD

**Discussion by aambadkar** [learndevopsonline/learndevopsonline#1515] [Q&A] [answered]
hi,
 
my peering connection is established but auto accept request is not working its seems pending for acceptance.
 
![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/bfd3b8b5-e63d-4eb9-a33a-3e6da6f586dc)
![Capture1](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/75a54538-4cb1-424c-b77b-bdbb5ee7bd28)
![Capture2](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/4dbdb8ec-b7c5-4ceb-b946-6f1d271d5047)

**Comment by sreedharm07** [comment, marked as answer]
Hi @aambadkar 
just  to check i rerunned your machine and i seen peering connection is working fine i dint did any changes. please re run it will work fine
